applications/GeoMechanicsApplication/custom_processes/deactivate_conditions_on_inactive_elements_process.hpp

```diff
@@ -62,9 +62,6 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) DeactivateConditionsOnInactiveElemen
     ///@name Operators
     ///@{
 
-    /// This operator is provided to call the process as a function and simply calls the Execute method.
-    void operator()() { Execute(); }
```

Comment:
This is exactly the same in the base process and it's not virtual so it was hiding that function